internal/cluster/clusterdriver/transport_http.go

```diff
@@ -42,6 +42,13 @@ func RegisterHTTPHandlers(endpoints Endpoints, router *mux.Router, options ...ki
 		options...,
 	))
 
+	router.Methods(http.MethodGet).Path("/nodepools").Handler(kithttp.NewServer(
+		endpoints.ListNodePools,
+		decodeListNodePoolsHTTPRequest,
+		kitxhttp.ErrorResponseEncoder(kitxhttp.StatusCodeResponseEncoder(http.StatusAccepted), errorEncoder),
```

Comment:
Shouldn't there be a response encoder as well? This will encode the endpoint response which is probably not the best option. Also, it should return OK, not Accepted.